src/order/filter/bases.lean

```diff
@@ -23,6 +42,10 @@ on some index set `ι`, and `s : ι → set α`.
   of bases.
 * `has_basis.tendsto_right_iff`, `has_basis.tendsto_left_iff`, `has_basis.tendsto_iff` : restate
   `tendsto f l l'` in terms of bases.
+* `is_countable_generated_iff_exists_antimono_basis` : proves a filter is
+  countably generated if and only if it admis a basis parametrized by a
+  decreasing sequence of sets.
```

Comment:
```suggestion
  decreasing sequence of sets indexed by `ℕ`.
```
(In this kind of generality, one might also think of sequences indexed by ordinals...)